examples/03_international_pricing/04_get_customer_orders/query.graphql

```diff
@@ -1,23 +1,23 @@
-query getcustomerOrders @inContext(country: FR) {
-  customer(customerAccessToken: "token") {
+query getcustomerOrders($customerAccessToken: String!, $country: CountryCode)@inContext(country: $country) {
+  customer(customerAccessToken: $customerAccessToken) {
     orders(first:10) {
       edges {
         node {
-          totalPriceV2 {
+          totalPrice {
             amount
-            currencyCode # order's currency - USD (point in time)
+            currencyCode # store's currency - CAD in this case
           }
           lineItems(first:10) {
             edges {
               node {
                 originalTotalPrice {
                   amount
-                  currencyCode # order's currency - USD (point in time)
+                  currencyCode # store's currency - CAD in this case
                 }
                 variant {
-                  priceV2 {
+                  price {
                     amount
-                    currencyCode # EUR variant's currency (past context)
+                    currencyCode # EUR currency (if French market is enabled for store)
```

Comment:
Same comment as above. Describing the field without calling out currencies. For example, localized currency